lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -21,6 +21,41 @@ def self.email(params = {}, _secrets = {}, _context = {})
 
         private_class_method def self.email_cleanup(*)
         end
+
+        def self.provision_execute(_params = {}, _secrets = {}, context = {})
+          return unless context.execution["_object_type"] == "ServiceTemplateProvisionTask"
+
+          ::MiqRequestTask.find(context.execution["_object_id"])&.execute
+
+          {"miq_request_task_id" => request_task_id}
+        end
+
+        private_class_method def self.provision_execute_status!(runner_context)
+          request_task = ::MiqRequestTask.find_by(:id => runner_context["miq_request_task_id"])
+          return if request_task.nil?
```

Comment:
Regardless, the caller does `method_result.merge(runner_context)`, so this will just cause the caller to blow up with a `undefined method `merge' for nil:NilClass`

Reply:
Right I'm thinking we should return `{"running" => false, "success" => "false", "output" => {"Error" => "States.TaskFailed", "Cause" => "Unable to find miq_request_task"}}` or something like that

Reply:
Oh no wait, I thought this was the execute method as opposed to the status method.  Ignore my comment.

Reply:
this nil is copied from email status.
I'll fix, but maybe we want to consider fixing that other one as well

Reply:
>  I'll fix, but maybe we want to consider fixing that other one as well

Yes I think I commented over in the PR that added that but didn't get to fix it, wanted to get it right here

Reply:
@kbrock https://github.com/ManageIQ/manageiq-providers-workflows/pull/87/files#diff-f49bf1f6a21ddda0e802d7717ef72ecaef1b11b86ee27fb5db029c7f770e3b0cR19